ui/App.svelte

```diff
@@ -102,10 +102,6 @@
 <NotificationFaucet />
 <Theme />
 
-{#if isExperimental && sessionIsUnsealed && $location !== path.designSystemGuide()}
-  <TransactionCenter />
```

Comment:
Could you also remove the corresponding `src/App/TransactionCenter.svelte` file?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1900/commits/78d594b8dde49394cc843f6f1a8b69c0faccfe65